src/i18n/locale/en.json

```diff
@@ -758,10 +761,11 @@
   "components.Settings.toastSettingsFailure": "Something went wrong while saving settings.",
   "components.Settings.toastSettingsSuccess": "Settings saved successfully!",
   "components.Settings.trustProxy": "Enable Proxy Support",
-  "components.Settings.trustProxyTip": "Allow Overseerr to correctly register client IP addresses behind a proxy (Overseerr must be reloaded for changes to take effect)",
```

Comment:
Why is this being removed?

Reply:
Oh i didnt rerun extract i18n after the last change you can see where it fudges up the line [here](https://github.com/sct/overseerr/pull/2383/files/63e4b0e60b079864aefef9728e8baf8761364417#diff-44dad73c083519cc1ee6cc31426a9c033cfd8ed9080b6ac8403bea720d807c72R368)